src/mlpack/methods/linear_svm/linear_svm_function.hpp

```diff
@@ -0,0 +1,210 @@
+/**
+ * @file linear_svm_function.hpp
+ * @author Shikhar Bhardwaj
+ * @author Ayush Chamoli
+ *
+ * Implementation of the hinge loss function for training a linear SVM with the
+ * parallel SGD algorithm.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_LINEAR_SVM_LINEAR_SVM_FUNCTION_HPP
+#define MLPACK_METHODS_LINEAR_SVM_LINEAR_SVM_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace svm {
```

Comment:
Picky comment, but can you leave an empty line between the namespace opening and the comment for `LinearSVMFunction`?